autosynth/synth.py

```diff
@@ -554,6 +554,7 @@ def _inner_main(temp_dir: str) -> int:
         "--repository", default=os.environ.get("REPOSITORY"), required=True
     )
     parser.add_argument("--synth-path", default=os.environ.get("SYNTH_PATH"))
+    parser.add_argument("--synth-file-name", default=os.environ.get("SYNTH_FILE_NAME"))
```

Comment:
We do this because we want to provide the full path (including the file name) to the synth file. `synth-path` makes autosynth `cd <synth-path>`, but we still want to run from the repo root.

Reply:
`synth-path` essentially runs `cd <synth-path> && python3 -m synthtool`. The `cwd` for the synthtool run is the `synth-path` directory.

Reply:
Added help text for t he command to clarify the options.